File: 3 КУРС/2 семестр/Учебная практика/JavaProects/app/src/main/java/com/example/voiceassistent/MessageEntity.java
package com.example.voiceassistent;

public class MessageEntity {
    public String text;
    public String date;
    public int isSend;

    public MessageEntity(String text, String  date, int isSend){
        this.text = text;
        this.date = date;
        this.isSend = isSend;
    }

    public MessageEntity(Message message){
        this.text = message.text;
        this.date = message.date.toString();
        this.isSend = (!message.isSend) ? 0 : 1;
    }

}

File: 3 КУРС/2 семестр/Учебная практика/JavaProects/app/src/main/java/com/example/voiceassistent/MessageEntity.kt
package com.example.voiceassistent

class MessageEntity {
    var text: String?
    var date: String
    var isSend: Int

    constructor(text: String?, date: String, isSend: Int) {
        this.text = text
        this.date = date
        this.isSend = isSend
    }

    constructor(message: Message) {
        text = message.text
        date = message.date.toString()
        isSend = if (!message.isSend) 0 else 1
    }
}